person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :name, :age, :id
+
+  def initialize(age, name: 'Unkown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] you added :id to the attr_accessor, this should only be added to the reader though.
- [ ] id should also be initialized as a random number.